.eslintrc.js

```diff
@@ -7,9 +7,6 @@ module.exports = {
     '@metamask/eslint-config',
     '@metamask/eslint-config/config/nodejs',
   ],
-  plugins: [
```

Comment:
While you're at it, it might be worth changing the `ecmaVersion` above to be `2017`, in-line with what the config uses.

Reply:
I'll do that!